gensim/test/utils.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+
+"""
+Common utils for tests
+"""
+import tempfile
+import os
+
+module_path = os.path.dirname(__file__)  # needed because sample data files are located in the same folder
+
+
+def datapath(fname):
```

Comment:
Please add docstring in [numpy-style](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) for both functions